cluster-autoscaler/cloudprovider/azure/azure_scale_set.go

```diff
@@ -254,9 +255,8 @@ func (scaleSet *ScaleSet) getScaleSetSize() (int64, error) {
 	size, err := scaleSet.getCurSize()
 	if size == -1 || err != nil {
 		klog.V(3).Infof("getScaleSetSize: either size is -1 (actual: %d) or error exists (actual err:%v)", size, err)
-		return size, err
 	}
-	return size, nil
+	return size, err
```

Comment:
This is just the same. Just removing the extra line of code, since we want to return `err` always (even if it's nil)